e2e-tests/cypress/tests/integration/channels/notifications/reset_notification_when_settings_canceled_spec.js

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @channels @notifications
+
+describe('Notifications', () => {
+    before(() => {
+        // # Upload and enable Calls plugin
+        cy.shouldNotRunOnCloudEdition();
+        cy.apiUpdateConfig({
+            PluginSettings: {
+                Enable: true,
+                EnableUploads: true,
+                PluginStates: {
+                    'com.mattermost.calls': {
+                        Enable: true,
+                        enableringing: true,
+                    },
+                },
+            },
+        });
+
+        cy.apiInitSetup().then(({team, user, channel}) => {
+            // # Login as user and visit channel
+            cy.apiLogin(user);
+            cy.visit(`/${team.name}/channels/${channel.name}`);
+        });
+    });
+
+    it('MM-53166 Notification sound modal selection should reset when settings canceled', () => {
```

Comment:
@dhnlr can you change the `MM-53166` to `MM-T5458`.
I've added a test in our test case management ([PR](https://github.com/mattermost/mattermost-test-management/pull/122/files))